server/routes/api/user.py

```diff
@@ -34,13 +35,14 @@
 
 
 class ImportStatus(Enum):
-    IMPORTED = 1
+    UPLOADED = 1
 
 
 @dataclass
 class Import:
     """Data type for import entry"""
-    status: int
+    status: int  # This is from ImportStatus Enum
+    files: List[str]
```

Comment:
nit: maybe `file_names` since files could be file handles. 

also would be great to document assumptions for these. looks like these are only file_names from an assumed fixed folder name. it might be worth storing the folder path too as part of the import so we can still recreate a full path to the files even if path assumptions change

Reply:
Change file name to an object with GCS bucket and object name now.